package org.kepler42.errors

class AlreadyRelatedException(message: String = "These entities are already related"): Exception(message)
